tutorials/asr/09_ASR_for_telephony_speech.ipynb

```diff
@@ -0,0 +1,423 @@
+    "\"\"\"\n",
+    "# exit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "v1Jk9etFlRzf"
+   },
+   "source": [
+    "# Telephony speech (8 kHz)\n",
+    "This notebook covers general recommendations for using NeMo models with 8 kHz speech. All the pretrained models currently available through NeMo are trained with audio at 16 kHz. This means that if the original audio was sampled at a different rate, it's sampling rate was converted to 16 kHz through upsampling or downsampling. One of the common applications for ASR is to recognize telephony speech which typically consists of speech sampled at 8 kHz.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mixed sample rate\n",
+    "Most of the pretrained English models distributed with NeMo are trained with mixed sample rate data, i.e. the training data typically consists of data sampled at both 8 kHz and 16 kHz. As an example pretrained Citrinet model \"stt_en_citrinet_1024\" was trained with the following datasets. \n",
+    "* Librispeech 960 hours of English speech\n",
+    "* Fisher Corpus\n",
+    "* Switchboard-1 Dataset\n",
+    "* WSJ-0 and WSJ-1\n",
+    "* National Speech Corpus - 1\n",
+    "* Mozilla Common Voice\n",
+    "\n",
+    "Among these, Fisher and Switchboard datasets are conversational telephone speech datasets with audio sampled at 8 kHz while the other datasets were originally sampled at at least 16 kHz. Before training all the audio from Fisher and Switchboard datasets Because of this mixed sample rate training our models can be used to recognize both narrowband (8kHz) and wideband speech (16kHz)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Inference with NeMo\n",
+    "NeMo ASR currently supports inference of audio in .wav format. Internally, the audio file is resampled to 16 kHz before inference is called on the model, so there is no difference running inference on 8 kHz audio compared to say 16 kHz or any other sampling rate audio with NeMo. Let's look at an example for running inference on 8 kHz audio. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is where the an4/ directory will be placed.\n",
+    "# Change this if you don't want the data to be extracted in the current directory.\n",
+    "data_dir = '.'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "******\n",
+      "Tarfile already exists.\n",
+      "Finished conversion.\n",
+      "******\n"
+     ]
+    }
+   ],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import subprocess\n",
+    "import tarfile\n",
+    "import wget\n",
+    "\n",
+    "# Download the dataset. This will take a few moments...\n",
```

Comment:
Maybe explain here that original sample rate is 16 kHz - or show it somewhere via print